website/docs/docs/collaborate/git/setup-azure.md

```diff
@@ -319,7 +324,8 @@ To connect the service user:
 
 Once connected, dbt Cloud displays the email address of the service user so you know which user's permissions are enabling headless actions in deployment environments. To change which account is connected, disconnect the profile in dbt Cloud, sign into the alternative Azure DevOps service account, and re-link the account in dbt Cloud.
 
-:::info Service user authentication expiration
-dbt Cloud will refresh the authentication for the service user on each run triggered by the scheduler, API, or CI. If your account does not have any active runs for over 90 days, an admin will need to manually refresh the authentication of the service user by disconnecting and reconnecting the service user's profile via the OAuth flow described above in order to resume headless interactions like project set up, deployment runs, and CI.
-
+:::info Personal Access Tokens (PATs)
+Full-scoped PATs are created within dbt Cloud for short access by the service user to hit APIs around dbt Cloud project setup. They are valid for only 5 minutes and are revoked after a call to the API has been made.
+The Azure DevOps Administrator in Azure AD may restrict users from creating full-scoped PATs. Enabling this policy means new PATs must be limited to a specific custom defined set of scopes. By default, this policy is set to off. If they do enable this, your project will throw an error. If using finer-scoped permissions, you will lose access to some features, like webhooks for CI.
```

Comment:
```suggestion
The Azure DevOps Administrator can limit the creation of full-scoped PATs by enabling a policy that restricts users to a custom-defined set of scopes. By default, this policy is set to **off**, but enabling it will cause the project to display an error. If you use finer-scoped permissions, some features such as webhooks for CI may be lost.

To restrict full-scoped PATs, add the dbt Cloud service user to the allow list to exclude that user from the global PAT policy as part of your security policy.
```